wellcomeml/datasets/conll.py

```diff
@@ -48,29 +53,35 @@ def _load_data_spacy(data_path, inc_outside=True):
                                 }
                             )
                         char_i += len(token) + 1  # plus 1 for the space separating
+
                 if sentence_tags != []:
                     X.append(sentence_text)
                     Y.append(sentence_tags)
 
     return X, Y
 
 
-def load_conll(split="train", shuffle=True, inc_outside=True):
-    path = check_cache_and_download("conll")
-
-    if split == "train":
-        train_data_path = os.path.join(path, "eng.train")
-        X, Y = _load_data_spacy(train_data_path, inc_outside=inc_outside)
-    elif split == "test":
-        test_data_path = os.path.join(path, "eng.testa")
-        X, Y = _load_data_spacy(test_data_path, inc_outside=inc_outside)
-    elif split == "evaluate":
-        eval_data_path = os.path.join(path, "eng.testb")
-        X, Y = _load_data_spacy(eval_data_path, inc_outside=inc_outside)
-    else:
-        raise ValueError(
-            f"Split argument {split} is not one of train, test or evaluate"
-        )
+def load_conll(split="train", shuffle=True, inc_outside=True, dataset: str = "conll"):
+    """Load the conll dataset
+
+    Args:
+        split(str): Which split of the data to collect, one of ["train", "test",
+            "evaluate"].
+        shuffle(bool): Should the data be shuffled with random.shuffle?
+        inc_outside(bool): Should outside charavters be included?
+        dataset(str): Which dataset to load. This defaults to "conll" and should
+            only be altered for test purposes in which case it should be set to
+            "test_conll".
+    """
+    path = check_cache_and_download(dataset)
+
+    map = {"train": "eng.train", "test": "eng.testa", "evaluate": "eng.testb"}
+
+    try:
+        data_path = os.path.join(path, map[split])
+        X, Y = _load_data_spacy(data_path, inc_outside=inc_outside)
+    except KeyError:
```

Comment:
This is the only lines being missed from the tests are these ones. I think you could very quickly add

```python
with pytest.raises(KeyError):
    load_conll(split='wrong_argument')
````

if you want to raise your coverage even more

Reply:
ooh nice :+1:

Reply:
Resolved in https://github.com/wellcometrust/WellcomeML/pull/154/commits/180fedf6e63d651b9ca482209114d678fe3d1547

Reply:
Cool, but now you got a flake8 pipeline problem :). After you fix that, and the tests pass, feel free to merge.